### PR TITLE
[uu-notify] add guide: uuNotify

### LINKED ISSUE #1058: add guide for uuNotify
**Background:** The more software I installed on my uberspace, I realized it's hard to keep it all up-to-date (checking for new releases, etc.), which is a risk.

**Solution:** A few weeks ago, I wrote some tiny helper script to notify me, if there are any updates available for software I installed on my uberspace. Since I installed most of the software I'm running on my uberspace based on instructions from the [lab](https://lab.uberspace.de/) I thought,  it might be of benefit for others, too.

So I decided to open source it (after I had cleaned it up a little 😉 ): https://github.com/franok/uu-notify

I would like to write a guide for _uuNotify_, so it's easier for other people on uberspace to keep their software up-to-date!
**EDIT: I'm currently working on this guide.**

_uuNotify_'s purpose is to notify a user (via gotify notifications), if there are updates available for a list of software they can maintain.

**Differentiation:** probably rss-feed-readers would do the same job, but this solution will provide a clean output, i.e. a short and descriptive notification about which software issued a new release (which version). So it's an alternative to RSS readers.

### PR DESCRIPTION
**Pull request by franok**
guide for https://github.com/franok/uu-notify

followed the CONTRIBUTING.md and STYLE.md guidelines

closes #1058

### CODE REVIEW

File: source/guide_uu-notify.rst [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+    [...]
+  }
+
+.. note:: The gotify server URL must end **without** trailing forward slash ``/``
+  
+  | Good example: ``https://isabell.uber.space/gotify``
+  | Bad example: ``https://isabell.uber.space/gotify/``
+
+
+Now add your `personal GitHub access token <https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token>`_. Example:
```
Comment (kimdiallo) [inline source/guide_uu-notify.rst:137]: A hint that the token only needs access to the notifications would be appropriate here.

**Review by kimdiallo — CHANGES_REQUESTED** [review]
Except for one small thing, the guide is so good. If you include the note there, I'll merge it that way.

**Comment by franok** [comment]
Thanks for the review. I'll be able to apply the suggested changes in approximately 4 weeks.

**Comment by nichtmax** [comment]
@franok since this is just a small change, would you be fine with us adding the hint? If you want to do it by yourself we can of course wait 4 weeks ☺️

**Comment by franok** [comment]
Yes, fine for me. Thanks :)

**Comment by nichtmax** [comment]
Thanks for your guide! Check your mail for your cup 🎉

**Pull request merged**